examples/build.gradle

```diff
@@ -22,7 +22,7 @@ targetCompatibility = 1.7
 
 // Feel free to delete the comment at the next line. It is just for safely
 // updating the version in our release process.
-def grpcVersion = '1.27.0-SNAPSHOT' // CURRENT_GRPC_VERSION
+def grpcVersion = '1.24.2' // CURRENT_GRPC_VERSION
```

Comment:
This will need to be reverted.

Reply:
Ah sry I accidentally pushed a bunch of random testing code In that last commit.  I'll get that cleaned up

Reply:
Done